Xabe.FFmpeg/Enums/VideoSize.cs

```diff
@@ -13,18 +13,36 @@ public class VideoSize
         /// </summary>
         internal readonly string Resolution;
 
+        /// <summary>
+        ///     FFmpeg resolution width
+        /// </summary>
+        internal readonly int Width;
+
+        /// <summary>
+        ///     FFmpeg resolution height
+        /// </summary>
+        internal readonly int Height;
+
         /// <summary>
         ///     VideoSize in FFmpeg format
         /// </summary>
         /// <param name="width">Width of output video</param>
         /// <param name="height">Height of output video</param>
         public VideoSize(int width, int height)
         {
+            Width = width;
+            Height = height;
             Resolution = $"{width}x{height}";
         }
 
```

Comment:
I would delete this method and leave only constructor with (int width, int height).
User shoudn't be worried about formatting resolution